.travis.yml

```diff
@@ -9,9 +9,11 @@ install:
   - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-ocaml.sh
   - bash -ex .travis-ocaml.sh
   - eval `opam config env`
+  - opam install -y ocp-indent
   - opam repository add satysfi-external https://github.com/gfngfn/satysfi-external-repo.git
   - opam update
   - opam pin add -y --no-action satysfi .
   - opam install -y --deps-only satysfi
 script:
+  - diff -u <(printf '') <(find src/ -name '*.ml' -or -name '*.mli' | xargs -I{} sh -c 'ocp-indent {} | diff -u {} -')
```

Comment:
or, more simply
```
- find src -name '*.ml' -o -name '*.mli' | xargs ocp-indent -i
- git diff --quiet
```

Reply:
I wanted to avoid writing files for performance gain, but I will measure it later.

Reply:
@leque Your method is actually faster than mine. I tested on WSL, so it may change on Linux on container but anyway it seems worth little. I will revise it. Thanks!